### PR TITLE
RAP-4141: added nightly series tests and fix to script runner class

### PR DESCRIPTION
**Pull request by jeremy-flieder**

### CODE REVIEW

File: Apps/RaptureIntTests/src/test/resources/local_testng.xml [hunk 1]
```diff
@@ -13,6 +13,9 @@
 			<package name="rapture.httpapi.series" />
 			<package name="rapture.httpapi.document" />
 			<package name="rapture.httpapi.plugin" />
+			<package name="rapture.nightly.java" />
```
Comment (jonathan-major) [inline Apps/RaptureIntTests/src/test/resources/local_testng.xml]: what is rapture.nightly.java?

Comment (jeremy-flieder) [inline Apps/RaptureIntTests/src/test/resources/local_testng.xml]: Currently it's where the java entitlement test lives. I will redo the packages if for no other reason than the java part is redundant... the tests are under src/java.

Comment (scarabus) [inline Apps/RaptureIntTests/src/test/resources/local_testng.xml]: a package name - we haven't formalized naming yet.
There's also rapture.nightly.java.search and rapture.nightly.reflex (stuff needed for Reflex testing)

Jeremy has been using rapture.httpapi.{apiname}

Comment (jonathan-major) [inline Apps/RaptureIntTests/src/test/resources/local_testng.xml]: can you agree on a package naming convention?

**Pull request merged**